nilearn/datasets/neurovault.py

```diff
@@ -2600,3 +2600,86 @@ def fetch_neurovault_ids(
         data_dir=data_dir,
         fetch_neurosynth_words=fetch_neurosynth_words,
         vectorize_words=vectorize_words, verbose=verbose)
+
+
+def fetch_neurovault_motor_task(data_dir=None, verbose=1):
+    """Fetch left vs right button press group contrast map from NeuroVault.
+
+    Parameters
+    ----------
+    data_dir: string, optional
+        Path of the data directory. Used to force data storage in a specified
+        location.
+
+    verbose: int, optional
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: Bunch         
+        A dict-like object which exposes its items as attributes. It contains:
+            - 'images', the paths to downloaded files.
+            - 'images_meta', the metadata for the images in a list of
+              dictionaries.
+            - 'collections_meta', the metadata for the
+              collections.
+            - 'description', a short description of the Neurovault dataset.
+
+    Notes
+    ------
+
+    This function is only a caller for the fetch_localizer_contrasts in order
+    to simplify examples reading and understanding.
+    The 'left vs right button press' contrast is used.
+
+    See Also
+    ---------
+    nilearn.datasets.fetch_neurovault_ids
+    nilearn.datasets.fetch_neurovault
+
+    """
+    data = fetch_neurovault_ids(image_ids=[10426], data_dir=data_dir,
+                                verbose=verbose)
+    return data
+
+
+def fetch_neurovault_auditory_computation_task(data_dir=None, verbose=1):
+    """Fetch a contrast map from NeuroVault showing
+    the effect of mental subtraction uopn auditory instructions
+
+    Parameters
+    ----------
+    data_dir: string, optional
+        Path of the data directory. Used to force data storage in a specified
+        location.
+
+    verbose: int, optional
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: Bunch         
+        A dict-like object which exposes its items as attributes. It contains:
+            - 'images', the paths to downloaded files.
+            - 'images_meta', the metadata for the images in a list of
+              dictionaries.
+            - 'collections_meta', the metadata for the
+              collections.
+            - 'description', a short description of the Neurovault dataset.
+
+    Notes
+    ------
+
+    This function is only a caller for the fetch_localizer_contrasts in order
+    to simplify examples reading and understanding.
```

Comment:
We should probably add a link to the neurovault web view of the corresponding image